concepts/05 UI Components/Gantt/20 Sort Data/Sort Data.md

```diff
@@ -0,0 +1,140 @@
+The Gantt component allows users to sort data. 
+
+Use the following properties to specify whether users can sort column data:
+
+- [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') - For all columns.
+- **column.**[mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) - For an individual column. Note that this property is in effect if the [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property is not set to 'none'.
+
+Use the **sorting**.[mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property to specify the sorting mode.
```

Comment:
"...to specify the sorting mode." - Does the sorting.mode property specify whether the Gantt component allows sorting data by multiple columns or by a single column only?